### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by chasballew**
@krallin - Some suggestions for the README, feel free to reject any/all. I think this is a really strong README, very well-written, and I don't have much to add. Great job!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Supercronic #
+# ![](https://raw.github.com/aptible/straptible/master/lib/straptible/rails/templates/public.api/icon-60px.png) Supercronic #
```
Comment (chasballew) [inline README.md]: Adding a logo is a quick way to increase awareness about Aptible without adding unnecessary text. We might consider a refreshed one with the "Aptible" mark that links.

File: README.md [hunk 2]
```diff
@@ -26,30 +26,27 @@ implementations are ill-suited for container environments:
...
+Finally, they are often quiet, making these issues difficult to understand and debug!
 
-The list could go on, but the fundamental takeaway is this: unlike typical
-server cron implementations, Supercronic tries very hard to do exactly what you
-expect from running `cron` in a container:
+Supercronic's goal is to behave how you would expect `cron` running in a container to behave:
 
-- Your environment variables are available in jobs.
-- Job output is logged to `stdout` / `stderr`.
+- Your environment variables are available in jobs
```
Comment (chasballew) [inline README.md]: Periods or no periods are fine either way, just aiming consistency within the list.

File: README.md [hunk 3]
```diff
@@ -26,30 +26,27 @@ implementations are ill-suited for container environments:
...
-- Job return codes and schedules are also logged to `stdout` / `stderr`.
+- Job return codes and schedules are also logged to `stdout` / `stderr`
 
 ## How does it work? ##
 
-- Install Supercronic (see below).
-- Point it at a crontab: `supercronic CRONTAB`.
+- Install Supercronic (see below)
+- Point it at a crontab: `supercronic CRONTAB`
 - You're done!
```
Comment (chasballew) [inline README.md]: This is an exception to the consistency rule. The rest of the listworks better without the periods (especially after the code block), making the exclamation point a nice little surprise at the end.

File: README.md [hunk 4]
```diff
@@ -94,13 +89,12 @@ does. In most cases, it should be compatible with your existing crontab.
 
 There are, however, a few exceptions:
 
-- First, Supercronic supports second-resolution schedules: under the hood,
+- First, Supercronic supports second-resolution schedules: Under the hood,
```
Comment (chasballew) [inline README.md]: Style guides vary on whether to use uppercase after a colon, but when the components can be broken into complete sentences, I think it reads better. Flame wars have been started over less, though. 🔥

**Comment by krallin** [comment]
Thanks! Made a couple markdown style fixes on top of your changes - you can see the diff here: https://github.com/chasballew/supercronic/compare/3b69cb8...fc6d894

**Pull request merged**